packages/sdk-tizen/src/deviceManager.ts

```diff
@@ -76,12 +76,13 @@ export const launchTizenSimulator = async (name: string | true): Promise<boolean
     logDefault(`launchTizenSimulator:${name}`);
 
     if (name === true) {
-        const targets = await execCLI(CLI_TIZEN_EMULATOR, 'list-vm', {
-            detached: true,
-        });
-        const lines = targets.split('\n');
-        const devicesArray = lines.map((line) => ({ id: line, name: line }));
-        const choices = _composeDevicesString(devicesArray);
+        const emulators = await execCLI(CLI_TIZEN_EMULATOR, 'list-vm');
+        const devices = await execCLI(CLI_SDB_TIZEN, 'devices');
+        const emulators_lines = emulators.split('\n');
+        const devices_lines = devices.split('\n');
+        const lines = emulators_lines.concat(devices_lines.slice(1));
```

Comment:
I'd say getting the first item here like this looks weird, but most likely there was a reason for it, so could you add a comment here?

Reply:
added comment to explain it